onebusaway-android/build.gradle

```diff
@@ -419,6 +419,7 @@ dependencies {
     implementation 'android.arch.lifecycle:common-java8:1.1.1'
     implementation 'androidx.constraintlayout:constraintlayout:2.0.0-alpha3'
     implementation 'commons-io:commons-io:2.4'
+    implementation 'org.apache.commons:commons-lang3:3.0'
```

Comment:
@barbeau I added Apache commons lang.  I used their HashCodeBuilder function.

Reply:
Ok - could we add regionId to the device properties too? I think we're want a quick way to group data by region, and this is property the easiest way.